wolfssl/internal.h

```diff
@@ -2352,7 +2352,8 @@ WOLFSSL_LOCAL void InitSuitesHashSigAlgo_ex(byte* hashSigAlgo, int haveECDSAsig,
                                             int haveRSAsig, int haveFalconSig,
                                             int haveDilithiumSig, int haveAnon,
                                             int tls1_2, int keySz, word16* len);
-WOLFSSL_LOCAL void InitSuitesHashSigAlgo_ex2(byte* hashSigAlgo, int have,
+/* use wolfSSL_API visibility to be able to test in tests/api.c */
+WOLFSSL_API void InitSuitesHashSigAlgo_ex2(byte* hashSigAlgo, int have,
```

Comment:
public API should not be in internal.h, They should also have our library prefix on them i.e wolfSSL_* or wc_*

Reply:
This is only marked public to allow for use inside `api.c`. It is not a real public API. I'm happy to explore other ways to export it to `api.c` if you have ideas.

Reply:
As soon as WOLFSSL_API gets added it shows up in the compiled library and we are signaling that it can be used by applications linking to wolfSSL. In the test case is it being used to more quickly setup the hash sig algo state rather than setting up a TLS connection?

Reply:
I see, was unaware that pattern was in place....I can understand that it helps with hitting those API in test cases. Looks like the precedent for the pattern was this PR (https://github.com/wolfSSL/wolfssl/pull/5503) version 5.5.1 of wolfSSL and on and then it was built upon.